libafl/src/fuzzer/mod.rs

```diff
@@ -470,6 +472,13 @@ where
                 let idx = state.corpus_mut().add(testcase)?;
                 self.scheduler_mut().on_add(state, idx)?;
 
+                self.num_testcases += 1;
```

Comment:
We don't really need to count here, `state.corpus().count()` should be the same number